### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by AlexandraOM** [fac18/week6-coda-squall#44]
Great range of tests and well done on getting Travis up and running too!

**Comment by georgiamshaw** [comment]
agreed! maybe add a few more in the main.test.js file though? appreciate this may be hard with the amount of DOM stuff in main.js, but see if you can strip things out and make the functions pure.
![Screenshot 2019-12-06 at 10 56 35 (2)](https://user-images.githubusercontent.com/36238916/70318102-25759a00-1817-11ea-8250-d371d735eefb.png)
